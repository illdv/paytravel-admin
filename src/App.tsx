import React, { SFC } from "react"
import { Admin, Resource } from "react-admin"
import jsonServerProvider from "ra-data-json-server"
import { UserList } from "components/Users"

const dataProvider = jsonServerProvider("http://jsonplaceholder.typicode.com")
const App: SFC<object> = () => (
  <Admin dataProvider={dataProvider}>
    <Resource name="users" list={UserList} />
  </Admin>
)

export default App
