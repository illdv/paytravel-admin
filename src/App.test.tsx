import React from "react"
import App from "./App"
import { shallow } from "enzyme"

it("renders without crashing", () => {
  const render = shallow(<App />)
  expect(render).toMatchSnapshot()
})
