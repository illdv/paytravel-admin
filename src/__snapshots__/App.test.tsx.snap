// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<withContext(CoreAdmin)
  appLayout={[Function]}
  catchAll={[Function]}
  dataProvider={[Function]}
  loading={[Function]}
  loginPage={[Function]}
  logoutButton={[Function]}
>
  <Connect(Resource)
    list={[Function]}
    name="users"
  />
</withContext(CoreAdmin)>
`;
